core/embed/sec/optiga/stm32/optiga_hal.c

```diff
@@ -38,6 +38,24 @@ void optiga_hal_init(void) {
   hal_delay(20);
 }
 
+void optiga_hal_deinit(void) {
+  GPIO_InitTypeDef GPIO_InitStructure = {0};
+
+  GPIO_InitStructure.Mode = GPIO_MODE_ANALOG;
```

Comment:
As setting pins as ANALOG when shutting down or going to low power is frequently repeating, maybe we could introduce some helper function for this, maybe in powerctl driver or introduce a gpio driver. Not very important, but it would save some typing and would be easier to understand.